things/postgres/channels.go

```diff
@@ -128,7 +128,7 @@ func (cr channelRepository) RetrieveAll(ctx context.Context, owner string, offse
 	}
 
 	q := fmt.Sprintf(`SELECT id, name, metadata FROM channels
-	      WHERE owner = :owner %s%s ORDER BY id LIMIT :limit OFFSET :offset;`, mq, nq)
+	      WHERE owner = :owner %s%s ORDER BY name LIMIT :limit OFFSET :offset;`, mq, nq)
```

Comment:
I am really not sure about this one... Is `name` even mandatory in our Thing entity?

Reply:
Name is not mandatory but sorting works for empty strings.